### PR TITLE
fix/publish-workflow

### PR DESCRIPTION
**Pull request by Oweeee**

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -31,8 +31,32 @@ jobs:
 
       - name: Publish
         env:
+          GIT_SSH_COMMAND: "ssh -o StrictHostKeyChecking=no"
           NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (bellstrand) [inline .github/workflows/publish.yml:35]: I'm missing the secrets for both `GITHUB_TOKEN ` & `GWENNIFER_GITHUB_SSH_KEY ` in this repositories `secrets`. We should add those before merging! :)

Comment (Oweeee) [reply .github/workflows/publish.yml:35]: Right! Do we just copy them from another repository, or are they stored somewhere else?

I also don't think I have access to the secret management in this repository

Comment (bellstrand) [reply .github/workflows/publish.yml:35]: Then I'll take a look at it after Christmas.
The are hashed in the other repositories, we have them stored in LastPass or in a Google Cloud Bucket.
remember me on Monday to fix it, they have also recently released "Organization Secrets" for bitbucket so you can use "global" secrets for all repositories which would be super nice.

Comment (Oweeee) [reply .github/workflows/publish.yml:35]: Oo, that would be nice. 
I'll poke you on monday so we can take a loot at it :)

**Pull request merged**